### PR TITLE
Update logging

### PR DESCRIPTION
**Pull request by MatthijsBurgh**

### CODE REVIEW

File: installer/tue-install-impl.bash [hunk 1]
```diff
@@ -75,14 +75,42 @@ function tue-install-debug
...
+
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+
+function tue-install-pipe
+{
+    local pipefail_old
+    pipefail_old=$(set -o | grep pipefail | awk '{printf $2}')
+    [ "$pipefail_old" != "on" ] && set -o pipefail # set pipefail if not yet set
+    echo -e "\e[0;1m[$TUE_INSTALL_CURRENT_TARGET]: $*\e[0m" | tee --append "$INSTALL_DETAILS_FILE"
+    "$@" 2> >(sed $'s,.*,\e[31m&\e[m,'>&1) | tee --append "$INSTALL_DETAILS_FILE"
```
Comment (ar13pit) [inline installer/tue-install-impl.bash:106]: Please add a comment about what is happening here, it takes a while to understand the logic

File: installer/tue-install-impl.bash [hunk 2]
```diff
@@ -1007,20 +1032,16 @@ function _tue-install-pip-now
 
     if [ -n "$pips_to_install" ]
     then
-        echo -e "Going to run the following command:\n"
-        echo -e "python${pv} -m pip install --use-feature=2020-resolver --use-feature=fast-deps --user $pips_to_install <<< yes\n"
         # shellcheck disable=SC2048,SC2086
-        python"${pv}" -m pip install --use-feature=2020-resolver --use-feature=fast-deps --user $pips_to_install <<< yes || tue-install-error "An error occurred while installing pip${pv} packages."
+        tue-install-pipe python"${pv}" -m pip install --use-feature=fast-deps --user $pips_to_install <<< yes || tue-install-error "An error occurred while installing pip${pv} packages."
```
Comment (ar13pit) [inline installer/tue-install-impl.bash:1036]: Also remove the --use-feature=fast-deps. The author of the feature himself said it doesn't work.

File: setup/tue-functions.bash [hunk 3]
```diff
@@ -620,13 +620,13 @@ function tue-revert
             if git -C "$pkg_dir"  diff -s --exit-code "$new_hash" "$current_hash"
             then
                 newtime=$(git -C "$pkg_dir"  show -s --format=%ci)
-                printf "\e[0;36m%-20s\033[0m %-15s \e[1m%s\033[0m %s\n" "$branch is fine" "$new_hash" "$newtime" "$pkg"
+                printf "\e[0;36m%-20s\e[0m %-15s \e[1m%s\e[0m %s\n" "$branch is fine" "$new_hash" "$newtime" "$pkg"
```
Comment (ar13pit) [inline setup/tue-functions.bash:623]: Do we need `printf` here?

**Review by ar13pit — CHANGES_REQUESTED** [review]

File: setup/tue-functions.bash (hunk 3 above)
Comment (MatthijsBurgh) [reply setup/tue-functions.bash:623]: This is not really in scope of this PR. So I keep it this way for now.

**Review by ar13pit — APPROVED** [review]

**Pull request merged**